lib/middleware/checkLicenceStatus.js

```diff
@@ -0,0 +1,42 @@
+const logger = require('../logger');
+const accessLicenceService = require('../services/accessLicenceService');
+const acquisitionContextService = require('../services/acquisitionContextService');
+
+
+module.exports = {
+
+    isActive :function* (req, res, next) {
+
+        try {
+
+            const currentLicence = yield accessLicenceService.getLicenceInfo(req.licenceId);
+
+            if (!currentLicence.status === 'active') {
```

Comment:
Is this supposed to be `!currentLicence.status` `===` `'active'` or `!(currentLicence.status === 'active')`? I'm guessing it's the second one, in which case it can be `currentLicence.status !== 'active'`.